pyproject.toml

```diff
@@ -1,6 +1,97 @@
 [build-system]
-requires = ["setuptools==62", "wheel"]
+requires = ["flit_scm"]
+build-backend = "flit_scm:buildapi"
+
+[project]
+name = "pydra"
+description = "Pydra dataflow engine"
+readme = "README.rst"
+requires-python = ">=3.7"
+dependencies = [
+    "attrs >=19.1.0",
+    "cloudpickle >=2.0.0",
+    "etelemetry >=0.2.2",
+    "filelock >=3.0.0",
+]
+license = {file = "LICENSE"}
+authors = [
+    {name = "Nipype developers", email = "neuroimaging@python.org"},
+]
+maintainers = [
+    {name = "Nipype developers", email = "neuroimaging@python.org"},
+]
+keywords = [
+    "brainweb",
+    "dataflow",
+    "neuroimaging",
+    "pydra",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+dask = [
+    "dask",
+    "distributed",
+]
+dev = [
+    "black",
+    "pre-commit",
+]
```

Comment:
```suggestion
dev = [
    "black",
    "pre-commit",
    "pydra[test]",
]
```

Reply:
MNT: Include test dependencies in dev group